apps/synapse-portal-framework/src/components/Explore/ExploreWrapper.tsx

```diff
@@ -62,6 +48,9 @@ export default function RouteControlWrapper(props: RouteControlWrapperProps) {
 
   return (
     <>
+      {explorePaths.map(route => (
+        <RouteMatchedOrientationBanner key={route.path} route={route} />
+      ))}
```

Comment:
Kind of a hacky solution to show an OrientationBanner on Explore pages, but it works for now.